loki/manifest.yaml

```diff
@@ -1,10 +1,10 @@
 ---
 name: loki-stack
 title: "Loki Stack "
-version: v0.41.2
+version: v2.1.2
 maintainer: "me@r15cookie.com"
 dependencies:
-  - prometheus-operator
+  - kube-prometheus-stack
```

Comment:
loki/manifest.yaml has a dependency on "kube-prometheus-stack" but that application doesn't exist to fix this error you have to use "prometheus-operator" as there is no app named as "kube-prometheus-stack" in marketplace and dependencies look for app in marketplace

Reply:
Thanks for taking a look!  This used to be set to "prometheus-operator", but my test scripts failed [had to update my test cluster to specify kube-prometheus-stack](https://github.com/ssmiller25/kubernetes-marketplace/blob/5acef324bf94ad016617cc090aec706adcdfd916/loki/Makefile#L68).  When I looked, it seems that promethues-operator isn't offered as an app, and only kube-prometheus-stack is available via the CLI.  I can move it back to prometheus-operator if that will work, but  confusing that I can't request that application to be installed by that name via the CLI. 

```sh
me@penguin:~/git/kubernetes-marketplace$ civo k3s application list | grep -i prometheus
| dynamic-pv-scaler         | 0.1.0            | storage      |                                                                                                                                              | prometheus-operator                 |
| kube-prometheus-stack     | 0.38.1           | monitoring   |                                                                                                                                              |                                     |
| loki-stack                | v0.41.2          | monitoring   |                                                                                                                                              | prometheus-operator                 |
```